pkg/nimble/Makefile

```diff
@@ -1,6 +1,6 @@
 PKG_NAME    = nimble
 PKG_URL     = https://github.com/apache/mynewt-nimble.git
-PKG_VERSION = a7b3c939146e735b59d55bff740c906bde6f86f9
+PKG_VERSION = 976bb8f84bf9547efdea444ae79a62d5a355a613
```

Comment:
If we merge #11273, this line should not be needed, right?

Reply:
Not sure, because https://github.com/apache/mynewt-nimble/commit/976bb8f84bf9547efdea444ae79a62d5a355a613 was merged this morning so it's more recent.

Reply:
True, didn't refresh my page. But the trouble with updating the version holds (see #11273). So I think it makes much sense to merge #11273 first.

Reply:
No problem, this is not urgent.